### ISSUE TITLE
Add support for units in nutritional plans

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#77]
## Use case

Add support for different units in the mobile app, you want to log "1 banana" and not "80g of banana"

## Proposal

The backend already supports this, we need to read the units and allow the user to set them when creating a plan or logging an ingredient

**Comment by M123-dev** [comment]
I see you are not using the [off-dart package](https://pub.dev/packages/openfoodfacts) but query from your backend. However, it is certainly interesting to know that we already have something like this in place.

https://github.com/openfoodfacts/openfoodfacts-dart/blob/c9f629bcf5e5ba11306587127f4998d9217ad130/lib/utils/UnitHelper.dart

**Comment by rolandgeider** [comment]
Yeah, I had my own DB structure before I knew of OFF! :) We will take a look at how you did it though

**Comment by rolandgeider** [comment]
this has been implemented in the meantime
